models/comments.models.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection");
+
+exports.selectCommentsById = (id) => {
+    return db 
+    .query(`
+    SELECT * FROM comments
+    WHERE article_id = $1;`, [id])
+    .then(({rows}) => {
+        if (rows.length === 0){
+            return Promise.reject({status: 404, msg: "Not Found" })
+        }
+        return rows[0];
+    })
+}
```

Comment:
You will need to add some extra logic to your model to pass the test I told you about 😄